docs/ref/request-response.txt

```diff
@@ -1304,7 +1304,7 @@ Attributes
     task to ``seek()`` it before passing it to ``FileResponse``.
 
     The ``Content-Length`` and ``Content-Type`` headers are automatically set
-    when they can be guessed from contents of ``open_file``.
+    from contents and ``open_file`` (or explicit filename) respectively.
```

Comment:
This gets even more confusing. I'd break it into two sentences:
```
    The ``Content-Length`` header is automatically set when it can be guessed
    from contents of ``open_file``.

    The ``Content-Type`` header is automatically set when it can be guessed
    from the ``filename``. If not provided, it's guessed from contents of
    ``open_file``.
```

Reply:
I think content type is not guessed from **contents** of ``open_file``, as the last sentence mention.
Content-type is guessed from file name of ``open_file`` too.

Reply:
> I think content type is not guessed from **contents** of `open_file`, as the last sentence mention. Content-type is guessed from file name of `open_file` too.

True, let's be precise:

```
    The ``Content-Length`` header is automatically set when it can be guessed
    from contents of ``open_file``.

    The ``Content-Type`` header is automatically set when it can be guessed
    from the ``filename`` or a name of ``open_file``.
```